RELEASE-NOTES.md

```diff
@@ -17,12 +17,12 @@ Packages released:
 ## Chore
 - **A11y:** Ensure SkipToContent sends focus to page content ([#2058](https://github.com/patternfly/patternfly-next/pull/2125))
 - **Build:**
-- Moved to circle ci. This PR:
- - Replaces .travis.yml with .circleci/config.yml and split up test steps to allow testing in parallel and nicer Github status checks
- - Fixes our our a11y Selenium script to not have Travis integrations
- - Uploads PR previews to https://surge.sh on every pushed commit (no need for Netlify to also build a preview)
- - Still updates the Github Pages with a new npm run publish:docs target on merges to master ([#2121](https://github.com/patternfly/patternfly-next/pull/212))
-- Temporarily disable problematic ci checks ([#2125](https://github.com/patternfly/patternfly-next/pull/2101))
+  - Moved to circle ci. This PR:
+  - Replaces .travis.yml with .circleci/config.yml and split up test steps to allow testing in parallel and nicer Github status checks
```

Comment:
Should these be sub-items of the line above? Or is it OK for them to be in the same hierarchy?

Reply:
What do you think makes the most sense? Right now it looks like it lives directly under Build.

Reply:
After I made the first edit, I think it resolves this comment. Let me know if it doesn't!